src/app/ui/timeline.cpp

```diff
@@ -181,13 +180,26 @@ Timeline::~Timeline()
 void Timeline::setZoom(double zoom)
 {
   m_zoom = MID(1.0, zoom, 10.0);
```

Comment:
Maybe here we can add a filter

    zoom = MID(1.0, zoom, 10.0);
    if (zoom == m_zoom)
      return;

    m_zoom = zoom;
    ...

But I'm not 100% sure.